src/components/organisms/ActionsPane/ActionsPane.styled.tsx

```diff
@@ -68,4 +70,29 @@ export const Tabs = styled(RawTabs)<{$height: number}>`
     display: flex;
     align-items: center;
   }
+
+  & .ant-tabs-tab {
```

Comment:
What happens if we are using tabs in multiple places under `ActionsPaneContainer`, I guess they will all get the same styling... Can we apply this only for the TitleBar?